### PR TITLE
Document Consul enterprise 1.10.0-1.10.4 forwards incompatibility with 1.11

### PR DESCRIPTION
**Pull request by mkeeler**
Also fixed a broken link in the 1.10.x upgrade instructions.

Live Docs Link to the new section: https://consul-oqyiiecv4-hashicorp.vercel.app/docs/upgrading/upgrade-specific#1-10-compatibility

### CODE REVIEW

File: website/content/docs/upgrading/upgrade-specific.mdx [hunk 1]
```diff
@@ -16,6 +16,14 @@ upgrade flow.
 
 ## Consul 1.11.0
 
+### 1.10 Compatibility <EnterpriseAlert inline />
+Consul Enterprise versions 1.10.0 through 1.10.4 contain a latent bug that
+causes those agents to deregister the Serf health check of the node when they
```
Comment (karl-cardenas-coding) [inline website/content/docs/upgrading/upgrade-specific.mdx:21]:
```suggestion
causes Consul client agents to deregister the Serf health check of the node when they 
```
Is this suggestion correct, Matt?

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Minor suggestion w/ question.

File: website/content/docs/upgrading/upgrade-specific.mdx (hunk 1 above)
Comment (mkeeler) [reply website/content/docs/upgrading/upgrade-specific.mdx:21]: Now that you mention it I _think_ it should just be client agents but am going to verify that with a test.

Comment (mkeeler) [reply website/content/docs/upgrading/upgrade-specific.mdx]: @karl-cardenas-coding Does this sound better to you?

File: website/content/docs/upgrading/upgrade-specific.mdx [hunk 2]
```diff
@@ -16,6 +16,14 @@ upgrade flow.
 
 ## Consul 1.11.0
 
+### 1.10 Compatibility <EnterpriseAlert inline />
+Consul Enterprise versions 1.10.0 through 1.10.4 contain a latent bug that
+causes those client or server agents to deregister their own services or health
+checks when some of the servers have been upgraded to 1.11. Prior
+to upgrading Consul Enterprise servers to 1.11, all Consul agents should first 
+be upgraded to 1.10.6 or higher to ensure forwards compatibility and to prevent
+flapping of catalog registrations.
```
Comment (karl-cardenas-coding) [inline website/content/docs/upgrading/upgrade-specific.mdx:25]:
```suggestion
Consul Enterprise versions 1.10.0 through 1.10.4 contain a latent bug that
causes those client or server agents to deregister their own services or health
checks when some of the servers have been upgraded to 1.11. Before upgrading Consul Enterprise servers to 1.11, all Consul agents should first 
be upgraded to 1.10.6 or higher to ensure forward compatibility and prevent
flapping of catalog registrations.
```

Looks great, just minor language optimization.

**Review by karl-cardenas-coding — APPROVED** [review]
Looks great, I just added a minor suggestion. 👍🏼

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/545489.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 7b5d52257e9d1cb819b66d854be27f3c58171f4f onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 7b5d52257e9d1cb819b66d854be27f3c58171f4f onto `release/1.11.x` succeeded!

**Pull request merged**